moto/mediastoredata/responses.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import unicode_literals
+
+import json
+
+from moto.core.responses import BaseResponse
+from .models import mediastoredata_backends
+
+
+class MediaStoreDataResponse(BaseResponse):
+    SERVICE_NAME = "mediastore-data"
+
+    @property
+    def mediastoredata_backend(self):
+        return mediastoredata_backends[self.region]
+
+    def get_object(self):
+        path = self._get_param("Path")
+        range = self._get_param("Range")
+        result = self.mediastoredata_backend.get_object(Path=path, Range=range)
+        return json.dumps(result.to_dict())
```

Comment:
AWS returns the actual content of the object as the body, but other parameters are returned as headers.
In moto, this can be done by returning a tuple:
```
headers = {"ETag": ..., "Cache-Control": "max-age=6, public", ...}
return "body of item", headers
```